framework/include/parser/CommandLine.h

```diff
@@ -116,6 +116,17 @@ class CommandLine
     return unused;
   }
 
+  enum class HitCmpResult
+  {
+    GLOBAL,
+    MAIN,
+    MATCH,
+    WRONG
```

Comment:
Maybe "MATCH" should be "MULTIAPP_MATCH", and "WRONG" should be "MULTIAPP_NOMATCH".